src/audio/pipeline/pipeline-schedule.c

```diff
@@ -113,9 +113,6 @@ void pipeline_schedule_triggered(struct pipeline_walk_context *ctx,
 {
 	struct list_item *tlist;
```

Comment:
@lgirdwood how did you get to this changes. Are there some tests or benchmarks that show performance drops with irqs disabled?

Reply:
This is to improve responsiveness for Zephyr scheduling where we expect to have IRQs of for minimal time (just like Linux).